utilities/src/apiSpecsList.json

```diff
@@ -2046,7 +2046,7 @@
     ]
   },
   "Microsoft.Cdn": {
-    "CdnWebApplicationFirewallPolicies": [
+    "cdnWebApplicationFirewallPolicies": [
```

Comment:
Updates to this file from PRs is something to look into. Do we have an issue for it already?

Reply:
We do, yes: https://github.com/Azure/ResourceModules/issues/2517